src/styles/_vars.scss

```
@@ -0,0 +1,15 @@
+$gap: 1px;
+$daySize: 100px;
+$padding: 160px;
+$min-days-calendar: 28;
+$max-days-calendar: 31;
+$dayBackground: #eee;
+$sizeCalendar: ($daySize * 6) + ($gap * 7);
+$days:
```

Хорошая работа, одна ошибочка
```suggestion
$dayBackground: #eee;
$sizeCalendar: ($daySize * 7) + ($gap * 7);
$days:
```
И вроде как все